examples/next-openai/app/function-calling/page.tsx

```diff
@@ -131,7 +131,7 @@ export default function Chat() {
 
   const { messages, input, handleInputChange, handleSubmit } = useChat({
     api: '/api/chat-with-functions',
-    onFunctionCall: functionCallHandler
+    experimentalOnFunctionCall: functionCallHandler
```

Comment:
specifically add `experimental_onFunctionCall` (with the underscore). This is how we do it at Vercel.